### PR TITLE
Chore: overview refactor image

### PR DESCRIPTION
**Pull request by lbc1013**
Ticket : https://trello.com/c/N2w39u12/135-overview-refactor-the-thumbnail-image-with-thumbnail-url-and-display-the-html-structure-while-loading-the-page

• Refactored the image to use thumbnail url.
• Refactored the unnecessary conditional rendering that I used for the test.

### CODE REVIEW

**Review by brynrmrzz — APPROVED** [review]
excited to see how these affect our changes

File: src/components/overview/AddToCart/SizeAndQuantitySelector.jsx [hunk 1]
```diff
@@ -48,39 +48,35 @@ const SizeAndQuantitySelector = ({state, updateState}) => {
...
+          {sizeAndQuantity !== null && (
+            Object.keys(sizeAndQuantity).map((element, index) => {
+              return <option id={`size${element}`} key={index}>{element}</option>;
+            })
+          )}
+        </select>
+        <select id='quantity'>
+          {quantity[0] !== undefined ?
+            quantity.map((element, index) => {
+              return <option id={`quantity${element}`} value={`quantity${element}`} key={index}>{element}</option>;
```
Comment (kjniemela) [inline src/components/overview/AddToCart/SizeAndQuantitySelector.jsx]: Possibly an excessively long line. Maybe split it?

File: src/components/overview/StyleSelector/StyleSelector.jsx [hunk 2]
```diff
@@ -53,31 +53,27 @@ const StyleSelector = ({state, updateState}) => {
...
-            </div>
+      <div id='styleSelector'>
+        <div id='styleTitle'>
+          STYLE : <b>{state.currentStyle.name}</b>
+        </div>
+        <div id='styleSelection'>
+          <div id='styleTop'>
+            {theFirstPhotoOfEachStyle.map((element, index) => {
+              if (index < 4) {
+                return <img onClick={updateCurrentStyle} className='eachStyleSelection' key={index} id={`style${index}`} src={element}></img>;
```
Comment (kjniemela) [inline src/components/overview/StyleSelector/StyleSelector.jsx]: Again, a very long line. Not sure if you think it'll be a problem, but I would have split it.

File: src/components/overview/AddToCart/SizeAndQuantitySelector.jsx (hunk 1 above)
Comment (lbc1013) [reply src/components/overview/AddToCart/SizeAndQuantitySelector.jsx]: I agree with you. I will split them for a better readability.

**Pull request merged**